src/loaders/FileLoader.js

```diff
@@ -77,7 +91,7 @@ class FileLoader extends Loader {
 		fetch( req )
 			.then( response => {
 
-				if ( response.status === 200 || response.status === 0 ) {
+				if ( response.status === 200 || response.status === 206 || response.status === 0 ) {
```

Comment:
I know its not new but I wonder what the 0 status is about.. Also again not new and maybe not worth changing but technically 200-299 are all "successful" responses.

Reply:
> I know its not new but I wonder what the 0 status is about..

See https://github.com/mrdoob/three.js/blob/f5e2d49247143b69778a9bbe0077041d5b92c37e/src/loaders/FileLoader.js#L93